web/packages/teleport/src/stores/storeUserContext.ts

```diff
@@ -191,4 +194,12 @@ export default class StoreUserContext extends Store<UserContext> {
   getAssistantAccess() {
     return this.state.acl.assist;
   }
+
+  getAccessCapabilities() {
+    return this.state.accessCapabilities;
+  }
```

Comment:
nit: we already have

https://github.com/gravitational/teleport/blob/17d4b45c345161174f45e3b04082a7c25791dc17/web/packages/teleport/src/stores/storeUserContext.ts#L79-L81

which is all this is being used for